theforeman.org/pipelines/release/source/smart-proxy.groovy

```diff
@@ -14,9 +14,9 @@ pipeline {
                 git url: git_url, branch: git_ref
             }
         }
-        stage("test ruby-2.7") {
+        stage("test ruby") {
```

Comment:
i've removed the 2.7 text from here because now we don't have a single ruby version to point to.